unitTests/ui/ResultLinkTest.ts

```diff
@@ -385,6 +397,27 @@ export function ResultLinkTest() {
         expect(test.cmp.element.getAttribute('href')).toEqual('');
       });
 
+      it(`when the uri uses the javascript protocol and contains a relative uri with a slash,
+        it clears the value to prevent XSS`, () => {
+        fakeResult.clickUri = 'javascript:"/casemgmt/sc_KnowledgeArticle"';
+        initHyperLink();
+        expect(test.cmp.element.getAttribute('href')).toEqual('');
+      });
+
+      it(`when the uri uses the javascript protocol and contains a relative uri with a period,
+        it clears the value to prevent XSS`, () => {
+        fakeResult.clickUri = 'javascript:"./articles/ka32C0000009t9CQAQ"';
+        initHyperLink();
+        expect(test.cmp.element.getAttribute('href')).toEqual('');
+      });
+
+      it(`when the uri uses the javascript protocol and contains a relative uri with two periods,
+        it clears the value to prevent XSS`, () => {
+        fakeResult.clickUri = 'javascript:"../articles/ka32C0000009t9CQAQ"';
+        initHyperLink();
+        expect(test.cmp.element.getAttribute('href')).toEqual('');
+      });
```

Comment:
If before adjusting the source these tests were failing, and after adjusting the source they started passing, then let's keep them. If they would have passed before modifying the source though, then they are not needed and can be removed.

Reply:
Without any of the code changes in this PR, they would've passed. However, with a easy mistake in the regex (which I originally did) they don't.